yarn.lock

```diff
@@ -1629,6 +1629,29 @@
     npmlog "^4.1.2"
     write-file-atomic "^2.3.0"
 
+"@loaders.gl/core@^2.2.3":
```

Comment:
Not sure it matters or is worth debugging at this stage, but should these resolve to 3.0.0-alpha on master?

Reply:
See `yarn list` output:
```
├─ @deck.gl/core@8.2.5
│  ├─ @loaders.gl/core@^2.2.3
│  ├─ @loaders.gl/core@2.3.6
│  │  ├─ @babel/runtime@^7.3.1
│  │  └─ @loaders.gl/loader-utils@2.3.6
│  ├─ @loaders.gl/images@^2.2.3
│  ├─ @loaders.gl/images@2.3.6
```
It comes from "@deck.gl/core" dependency.

Reply:
I see, that explains it. 

If it becomes an issue later, we can add a "resolutions" field in top-level package.json to force a specific version.